### PR TITLE
fw-4781, adjust site permissions

### PR DESCRIPTION
**Pull request by sarahfirstvoices**
### Description of Changes
- changed model permissions to match ability to access site details (e.g., members can see member site, team members can see team site)
- site list api has custom handling to show summary info for member + public sites

### Checklist
- [ ] README / documentation has been updated if needed
- [ ] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable
- [ ] Admin Panel has been updated if models have been added or modified
- [ ] Migrations have been updated and committed if applicable
- [ ] Team Postman workspace has been updated if applicable

### Reviewers Should Do The Following:
- [ ] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/backend/views/sites_views.py [hunk 1]
```diff
@@ -107,7 +114,7 @@ def list(self, request, *args, **kwargs):
         Return a list of sites grouped by language.
         """
         # retrieve visible sites in order to filter out empty languages
-        sites = Site.objects.visible(self.request.user)
+        sites = Site.objects.filter(visibility__gte=Visibility.MEMBERS)
```
Comment (NolanVH) [inline firstvoices/backend/views/sites_views.py:117]: Will this change mean that a language admin will not see a team site that they belong to on the explore languages page?

**Review by NolanVH — APPROVED** [review]
Aside from the one small question I have, all changes look good to me!

File: firstvoices/backend/views/sites_views.py (hunk 1 above)
Comment (sarahfirstvoices) [reply firstvoices/backend/views/sites_views.py:117]: Yes, correct. Intentionally

Comment (sarahfirstvoices) [reply firstvoices/backend/views/sites_views.py:117]: It still shows in the "my sites" section at the top of the page

**Pull request merged**